tutor/src/screens/student-dashboard/event-row.jsx

```diff
@@ -105,7 +109,7 @@ class EventRow extends React.Component {
             <span>{event.title}</span>
           </TitleCell>
           <DueCell>
-            <EventTime event={event} />
+            <EventTime event={event} course={course} />
             {event.is_extended && <StyledEIcon />}
```

Comment:
course doesn't seem to be used in EventTime

Reply:
Oops good catch, forgot I changed course there. Weirdly, lint didn't tell me!